samples/bluetooth/broadcast_assistant/boards/nrf52840dk_nrf52840.conf

```diff
@@ -0,0 +1 @@
+CONFIG_BT_CTLR_SCAN_DATA_LEN_MAX=191
```

Comment:
is it really board-specific? there seems to be a few samples (ex. `encrypted_advertising/central`) setting this in prj.conf.

Reply:
I guess we could move it to the prj.conf - I'll do that